source/ways-of-working/oat/index.html.md.erb

```diff
@@ -0,0 +1,189 @@
+
+### Error Handling Framework
+
+This OAT is to ensure facilities are in place for errors/exceptions to recorded to be referred against in case of production issues.
+
+- SOG to be updated with list of all exceptions being raised by the service with possible cause and resolution marked against.
+- SOG should also confirm adherence to [status code and error standards](https://hmcts.github.io/restful-api-standards/#http-status-codes-and-errors)
+
+### Data Retention
+
+This OAT is to ensure that this release complies with data retention policies set by HMCTS and GDPR.
+
+- SOG should outline data retention policy on all the data being stored by the services.
+- This is applicable to all the direct data stores that services use like Redis, PostgreSQL and any data that is stored in dependent systems like CCD.
+- Application should also make sure that data should not be reintroduced on restoring from backup.
+
+### Service Continuity
+
+This OAT is to ensure that the service has a robust approach for dealing with the event of a catastrophic failure.
+
+- Application team to make sure they have `/health/liveness` URLs configured so that Kubernetes can automatically restart the service for restoration.
+- Make sure any cron jobs that are not running on services that are serving end user traffic.
+- Elaborate in SOG on how Jobs would handle failures.
+- SOG should add reasoning if using higher CPU/Memory requests than defaults.
+
+### Shuttering
+
+This OAT is applicable for front end services to ensure they have shuttering mechanism in place.
+
+- [Shuttering](https://hmcts.github.io/ways-of-working/path-to-live/shutter.html) should be configured and tested before documenting in SOG.
+
+### Resilience / Recoverability
+
+This OAT is to ensure that the system can recover or fail over to a secondary component in the event of failure.
+
+- All services should be configured with appropriate `readiness` and `liveness` checks.
+- Services deployed through flux, automatically get deployed to multiple clusters with multiple instances running.
+- Development teams to confirm that all services (including jobs) are configured to run in both clusters.
+- Performance test results should be attached to the task.
+- Development team to make sure any performance tuning changes are applied to prod.
+- Any services running scheduled jobs should update SOG on how they handle failures and if running multiple jobs at same time can cause any issues.
+
+### Reliability
+
+This OAT is to ensure that the service is reliable which is measured by Mean Time Between Failures (MTBF).
+
+- Unless specially called out, all apps deployed to AKS using Flux should be compliant with the standard platform reliability.
+
+### Data Backup & Recovery
+
+This OAT is to ensure the data can be reliably and frequently backed up.
+
+- Development team should confirm that they adhere to [PostgreSQL standards](https://tools.hmcts.net/confluence/display/RD/DBaaS+PostgreSQL+Standards)
```

Comment:
shouldn't really link off to an outdated confluence page like this.

It walks about manual locks, the devops team and read replicas that aren't in use

Reply:
Referred it to the relevant subsection. Will ask Alliu to update the rest of the document.